x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/alpha_messaging.tsx

```diff
@@ -28,14 +28,14 @@ export const AlphaMessaging: React.FC<{}> = () => {
           <strong>
             <FormattedMessage
               id="xpack.ingestManager.alphaMessageTitle"
-              defaultMessage="Experimental"
+              defaultMessage="Beta release"
             />
           </strong>
           {' – '}
           <FormattedMessage
             id="xpack.ingestManager.alphaMessageDescription"
             defaultMessage="Ingest Manager is under active development and is not
-          intended for production purposes."
+          intended for use in production environments."
           />{' '}
           <EuiLink color="subdued" onClick={() => setIsAlphaFlyoutOpen(true)}>
             View more details.
```

Comment:
I happened to notice that this string isn't wrapped in `FormattedMessage`, any chance you can fix it in this PR? 🙏

Reply:
yup i noticed too :). Updated!